### ISSUE TITLE
VoiceOver doesn't read full quick pick entry if it has a codicon

### ISSUE THREAD

**Issue by ejizba** [microsoft/vscode#120992]
Issue Type: <b>Bug</b>

_originally filed in https://github.com/microsoft/vscode-azurefunctions/issues/2687_

## Repro Steps

1. Start with a standard hello world extension
2. Add this code to the default "Hello World" command
    ```typescript
    await vscode.window.showQuickPick(['$(squirrel) hello', '$(squirrel)$(squirrel)$(squirrel) hello'], {
        ignoreFocusOut: true,
        placeHolder: 'Hello world'
    });
    ```
3. F5 the extension
4. Turn on VoiceOver for Mac
5. Run the "Hello World" command
6. Move keyboard focus to the first quick pick entry

## Expected

It reads "Hello"

## Actual

It reads "hell"
<img width="426" alt="Screen Shot 2021-04-09 at 4 43 57 PM" src="https://user-images.githubusercontent.com/11282622/114250672-56a63300-9953-11eb-8fec-050303d5b698.png">

My suspicion was the the codicon was messing with it, so I confirmed if I use more codicons, it cuts off more of the message
<img width="335" alt="Screen Shot 2021-04-09 at 4 43 52 PM" src="https://user-images.githubusercontent.com/11282622/114250702-6887d600-9953-11eb-91e2-b1c610bcb865.png">

VS Code version: Code - Insiders 1.56.0-insider (845efe5d8e31079cc858131c95c36103d9439e74, 2021-04-09T09:50:42.652Z)
OS version: Darwin x64 20.3.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz (12 x 2600)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|4, 3, 3|
|Memory (System)|16.00GB (1.07GB free)|
|Process Argv|--crash-reporter-id 63154ab5-ba0e-419a-aa36-bd67d86db27f|
|Screen Reader|no|
|VM|0%|
</details>

**Comment by TylerLeonhardt** [comment]
For endgame verification:

The repro steps are already great verification steps.

**Comment by TylerLeonhardt** [comment]
Good to know. Thanks @isidorn! 🐿
